recipes/fswatch/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./configure --prefix="$PREFIX"
+make
+make check
```

Comment:
Added this, but don't think it is doing anything. Raised issue ( https://github.com/emcrisostomo/fswatch/issues/198 ) to see how to run the tests correctly.